find-index.js

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable indent */
+const findIndex = (arr, callback) => {
+    
+    for (let index = 0; index < arr.length; index++) {
+        const item = arr[index];
+        if(callback(item) === true) {
+        return arr[index];
```

Comment:
I'm pretty sure if you change this line to `return index` this will give the result the function is intended for (and which should make updated tests pass)